### DISCUSSION TITLE
Problem with signaling server/High-perfomance backend

### DISCUSSION THREAD

**Discussion by Fred-DTV** [nextcloud/all-in-one#1847] [Questions] [answered]
I am having a weird issue with the Talk High-performance backend in my AIO installation.

The standard link which is inserted (and I guess is supposed to direct to an internal signaling server) doesn't work and it stops the whole Talk app from being able to send and receive messages.
However, when I delete the link, everything works fine, but unfortunately with every daily backup the standard link is inserted again and it it stops Talk from working again.
This is the URL which is being inserted: https://my.sub.domain.live/standalone-signaling/

I have a valid SSL certificate and opened the ports 3478 TCP+UDP
This is the error message form the log:
https://pastebin.com/xBaa6Znm

The simplest solution for me would be to stop the link from appearing again after each backup, but I don't know how to do that...

P.S.: And I can't leave it empty either, like the descriptions says. It will just have the standard URL after the next reboot again

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by Fred-DTV** [reply]
Thanks. Would you prefer i post the result there or here?

**Reply by szaimen** [reply]
yes, please do so

**Comment by szaimen** [comment, marked as answer]
Will be handled in https://github.com/nextcloud/all-in-one/discussions/1358#discussioncomment-4804762
